lib/Dialect/SV/SVOps.cpp

```diff
@@ -212,7 +211,7 @@ void LocalParamOp::getAsmResultNames(OpAsmSetValueNameFn setNameFn) {
 LogicalResult LocalParamOp::verify() {
   // Verify that this is a valid parameter value.
   return hw::checkParameterInContext(
-      value(), (*this)->getParentOfType<hw::HWModuleOp>(), (*this));
+      value(), (*this)->getParentOfType<hw::HWModuleOp>(), *this);
```

Comment:
Supreme-nit: always feel free to land these types of fixes separately. (Not a blocker.)